platform/frontend/src/app/settings/appearance/page.tsx

```diff
@@ -33,8 +33,7 @@ export default function AppearanceSettingsPage() {
     setPreviewFont,
     applyThemeOnUI,
     applyFontOnUI,
-    saveTheme,
-    saveFont,
+    saveBoth,
```

Comment:
are `saveTheme` and `saveFont` used anywhere now? If not, I would recommend remove them from the definition in `useOrgTheme` and in addition, rename `saveBoth` to `saveAppearance`